syntax = "proto2";
package cardgame;

import "struct.proto";
option optimize_for = LITE_RUNTIME;
option java_package = "com.zhili.cardgame.proto";  
option java_outer_classname = "Bank";


message QueryBankReq {
	enum DEF {
		ID = 4500;
	}
	
}

message QueryBankResp {
	enum DEF {
		ID = 4501;
	}
	optional int64 gold = 1;
	optional int64 limit = 2;
	optional int64 next_vip_limit = 3;
}

message SaveMoneyReq {
	enum DEF {
		ID = 4502;
	}
	required int64 gold = 1;
}

message SaveMoneyResp {
	enum DEF {
		ID = 4503;
	}
	required Result result = 1;
}

	

